.github/workflows/rust.yml

```diff
@@ -67,49 +69,16 @@ jobs:
       working-directory: ${{ github.workspace }}/lurk
```

Comment:
We can also use the https://github.com/lurk-lab/ci-workflows/blob/main/.github/workflows/check-lurk-compiles.yml workflow here, or wait till it's more versatile as it only accepts a `path` and optional `package` keyword for the patch

Reply:
> wait till it's more versatile as it only accepts a path and optional package keyword for the patch

I'm not sure I understand what you're suggesting?

Reply:
Future work could be to refine the `check-lurk-compiles` reusable workflow to account for things like arecibo having a `package` Cargo rename, different features, etc. It's not related to this PR